README.md

```diff
@@ -98,3 +98,16 @@ Plugins are very welcome to extend Bevy's features. [Guidelines][plugin_guidelin
 ## Thanks and Alternatives
 
 Additionally, we would like to thank the [Amethyst](https://github.com/amethyst/amethyst), [macroquad](https://github.com/not-fl3/macroquad), [coffee](https://github.com/hecrj/coffee), [ggez](https://github.com/ggez/ggez), [rg3d](https://github.com/mrDIMAS/rg3d), and [Piston](https://github.com/PistonDevelopers/piston) projects for providing solid examples of game engine development in Rust. If you are looking for a Rust game engine, it is worth considering all of your options. Each engine has different design goals, and some will likely resonate with you more than others.
+
+## License
+
+Bevy is free and open source! It is dual-licensed under either:
```

Comment:
"It" in this case *could* be more clear/specific, e.g. All code in this repository, etc.

It might be worth noting that not all of our dependencies are under MIT/Apache 2.0, for example, and pointing people towards [cargo deny](https://github.com/EmbarkStudios/cargo-deny/). We could do with trying to get as many as possible under it though (e.g. `notify`'s license is *slightly* annoying in that it isn't recommended to use it for code)

On the other hand, those notes would probably be better for a page on the website (about distributing your game, for example). We should point to that from here though, when that exists (e.g. `distributing#license-compliance` - we shouldn't claim to be definitive on this page, but make sure that people are aware that licenses exist)

Speaking of which, we should relicense the website too.

Reply:
> "It" in this case could be more clear/specific, e.g. All code in this repository, etc.

Good call!

> It might be worth noting that not all of our dependencies are under MIT/Apache 2.0, for example, and pointing people towards cargo deny. We could do with trying to get as many as possible under it though (e.g. notify's license is slightly annoying in that it isn't recommended to use it for code)

I'd prefer to not cover this in the readme to avoid overwhelming the user with information. The dual license already complicates things more than I'd like. We definitely need a license-compliance page on the website and I think that is the right place for this sort of thing.

> Speaking of which, we should relicense the website too.

*\*dies inside\**

>

Reply:
> Speaking of which, we should relicense the website too.

It is definitely worth discussing. But the dual license doesn't really yield the same fruits in that context. The only real functional reason that I can think of is "consistency". But I care less about that relative to managing yet another relicensing effort.

Reply:
Yeah, exactly. I'm more just noting that we could and maybe should link to the license compliance page from this section of the readme, once it exists.